keras_retinanet/bin/train.py

```diff
@@ -315,10 +328,14 @@ def main(args=None):
         weights = args.weights
         # default to imagenet if nothing else is specified
         if weights is None and args.imagenet_weights:
-            weights = download_imagenet(RESNET_BACKBONE)
+            weights = download_imagenet(args.backbone)
 
         print('Creating model, this may take a second...')
-        model, training_model, prediction_model = create_models(num_classes=train_generator.num_classes(), weights=weights, multi_gpu=args.multi_gpu)
+        model, training_model, prediction_model = create_models(backbone_retinanet=retinanet,
```

Comment:
Reshape as:
```python
foo(
    bar,
    baz
)
```

Reply:
model, training_model, prediction_model = create_models(backbone_retinanet=retinanet,
                                                                backbone=args.backbone,
                                                                num_classes=train_generator.num_classes(),
                                                                weights=weights,
                                                                multi_gpu=args.multi_gpu)  
=>
      model, training_model, prediction_model = create_models(backbone_retinanet=retinanet, backbone=args.backbone, num_classes=train_generator.num_classes(), weights=weights, multi_gpu=args.multi_gpu)

?

Reply:
```python
      model, training_model, prediction_model = create_models(backbone_retinanet=retinanet,
                                                                backbone=args.backbone,
                                                                num_classes=train_generator.num_classes(),
                                                                weights=weights,
                                                                multi_gpu=args.multi_gpu)  
```
to:
```python
model, training_model, prediction_model = create_models(
    backbone_retinanet=retinanet,
    backbone=args.backbone,
    num_classes=train_generator.num_classes(),
    weights=weights,
    multi_gpu=args.multi_gpu
)
```